crates/basilica-sdk-python/python/basilica/__init__.py

```
@@ -260,6 +266,8 @@ def _extract_gpu_model_id(full_name: str) -> str:
     "StopCpuRentalResponse",
     "CpuRentalListItem",
     "ListCpuRentalsResponse",
+    # GPU Rental types (secure cloud)
+    "GpuOffering",
 ]
```

Ruff (0.14.13)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)